code/_macros.dm

```diff
@@ -121,4 +121,10 @@
 
 #define OPPOSITE_DIR(D) turn(D, 180)
 #define TURN_LEFT(D) turn(D, 90)
-#define TURN_RIGHT(D) turn(D, -90)
\ No newline at end of file
+#define TURN_RIGHT(D) turn(D, -90)
+
+
+#define SPAN(class, X)   "<span class='[class]'>[X]</span>"
```

Comment:
Careful, you have a few spaces here, just needs one

Reply:
I think it would make this part messier then, and would become even more if someone would like to add more stylesheet macroses. It works like this anyways